src/Icons/src/IconRenderer.php

```diff
@@ -36,11 +42,16 @@ public function __construct(
      */
     public function renderIcon(string $name, array $attributes = []): string
     {
-        $name = $this->iconAliases[$name] ?? $name;
+        $iconName = $this->iconAliases[$name] ?? $name;
 
-        $icon = $this->registry->get($name)
-            ->withAttributes($this->defaultIconAttributes)
-            ->withAttributes($attributes);
+        $icon = $this->registry->get($iconName);
+
+        if (0 < (int) $pos = strpos($name, ':')) {
```

Comment:
```suggestion
        if (false !== $pos = strpos($name, ':')) {
```

Reply:
No, this avoid me a check if the passed icon name start with a ":" ... and thus ensure the prefix is not empty ;)